front/src/components/recovery-password/mmt-recovery-step-1.vue

```diff
@@ -0,0 +1,11 @@
+<template src='./mmt-recovery-step-1.html'>
+
+</template>
+
+<script src='./mmt-recovery-step-1.js'>
+
+</script>
+
+<style scoped src="./mmt-recovery-step.css">
```

Comment:
why css without step numbers?

Reply:
because there are only two styles and they only touch two steps. the rest is governed by the built-in classes